notebooks/outetts-text-to-speech/outetts-text-to-speech.ipynb

```diff
@@ -22,6 +22,11 @@
     "- [Run model inference](#Run-model-inference)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #36.                "__module.model.layers.*.self_attn/aten::scaled_dot_product_attention/ScaledDotProductAttention"
</pre><p>Could you please provide a reason in the description for why we should add this pattern to <code>IgnoredScope</code>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2662/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Done.